vs/windows.undocked.props

```diff
@@ -17,6 +17,9 @@
     <UndockedBuildArm64x Condition="'$(UndockedBuildArm64x)' == '' OR $(ARM64X_DISABLED) == '1'">false</UndockedBuildArm64x>
     <!-- Use the official LKG complier when available -->
     <UseInternalMSUniCrtPackage>true</UseInternalMSUniCrtPackage>
+    <UndockedPlatConfig Condition="'$(UndockedPlatConfig)' == ''">$(Platform)_$(Configuration)</UndockedPlatConfig>
```

Comment:
`Platform` and `Configuration` are defined below.

Reply:
hmm, it seems to work nevertheless, but maybe it would break for Visual Studio.